client/state/ui/reducer.js

```diff
@@ -79,7 +79,7 @@ export function hasSidebar( state = true, action ) {
 		case SERIALIZE:
 			return true;
```

Comment:
I'm still working on a good validation strategy in #3101, but if it makes sense to persist state, feel free to serialize it as well. So perhaps ``` return state; ``` here and in `section`